test/gov/clean_css.sh

```diff
@@ -14,5 +14,18 @@ hzn mms object delete -t "model" -i "multires.tgz"
 hzn mms object delete -t "model" -i "policy-basicres.tgz"
 hzn mms object delete -t "model" -i "policy-multires.tgz"
 sleep 5
+hzn mms object delete -t "test" -i "test-medium1"
```

Comment:
Just a general question about this class: where is this script called? I didn't find the place. Are we still using this class?

Reply:
Good catch, I just did a search for where we were using `hzn mms object` and this was one of the files that came back. I didn't think to check where it's being called from and it doesn't appear to be used anywhere. I did use it to clean things up when I was testing though.